crates/sb_graph/lib.rs

```diff
@@ -67,7 +69,13 @@ pub async fn generate_binary_eszip(
             String::from_utf8(entry_content.clone())?.into()
         };
         let emit_source = emitter_factory.emitter().unwrap().emit_parsed_source(
-            &ModuleSpecifier::parse("http://localhost").unwrap(),
+            &ModuleSpecifier::parse(
+                &*Url::from_file_path(&fs_path)
+                    .map(|it| Cow::Owned(it.to_string()))
+                    .ok()
+                    .unwrap_or("http://localhost".into()),
```

Comment:
Is there I've modified this one for source mapping because not to this, the inspector can't be mapped with the host script correctly.

Is there a specific reason why this module specifier must be localhost?

Reply:
cc: @andreespirela does it need to be `localhost` here?